lib/improver/utilities/cube_manipulation_new.py

```diff
@@ -314,367 +316,417 @@ def concatenate_cubes(
     return result
 
 
-def merge_cubes(cubes, model_id_attr=None, blend_coord=None):
+class MergeCubes():
     """
-    Function to merge cubes, accounting for differences in the
-    attributes, and coords.
+    Class adding functionality to iris.merge_cubes()
 
-    Args:
-        cubes (Iris cubelist or Iris cube):
-            Cubes to be merged.
+    Accounts for differences in attributes and coordinates to avoid merge
+    failures and anonymous dimensions.
+    """
+    def __init__(self):
```

Comment:
Why are these not passed in? What is the rational for silently ignoring these and the coord_mismatch_error_keys as well?